Apps/Playground/Scripts/experience.js

```diff
@@ -51,6 +51,8 @@ function CreateInputHandling(scene) {
     });
 }
 
+BABYLON.Tools.DelayAsync(1000).then(() => {
```

Comment:
This is obvious nonsense, but it's filling in for a real API. I don't think this can be a part of what gets checked in, but we should figure out whether we want to go all the way to a real solution (if we know what one is at this point) or find a more sustainable stopgap to go in with this change as an intermediate.